lib/babe/babe.go

```diff
@@ -185,7 +193,7 @@ func (b *Session) Descriptor() *NextEpochDescriptor {
 func (b *Session) safeSend(msg types.Block) error {
 	b.lock.Lock()
 	defer b.lock.Unlock()
-	if b.closed {
+	if atomic.LoadUint32(&b.closed) == uint32(0) {
```

Comment:
also update here if you change the variable name: `b.stopped == uint32(1)`

Reply:
I updated this to started, please re-check.